modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/services/GlobalL0Service.scala

```diff
@@ -34,6 +34,7 @@ import org.typelevel.log4cats.slf4j.Slf4jLogger
 trait GlobalL0Service[F[_]] {
   type LatestSnapshotTuple = (Hashed[GlobalIncrementalSnapshot], GlobalSnapshotInfo)
   def pullLatestSnapshot: F[LatestSnapshotTuple]
+  def pullLatestSnapshotFromRandomPeer: F[LatestSnapshotTuple]
```

Comment:
if you add `pullLatestSnapshotFromRandomPeer` to the trait itself, it makes me thinking what `pullLatestSnapshot` does. IMO it makes sense to either change the `pullLatestSnapshot` name as well, or parametrize the function instead of creating next one, like:

```scala
def pullLatestSnapshot(useRandomPeer: Boolean): ...
```

Reply:
@marcinwadon I initially did that but previous PR comment suggested to make the call explicit instead

Reply:
`pullLatestSnapshot` uses random or majority based on the configuration, we don't know the configuration at the call site. On the other hand we want to explicitly pull from random peer here.

Reply:
At the same time `useRandomPeer` sounds much better than `skipMajorityCheck` so we could go with such a param I think if you guys like that solution more. `skipMajorityCheck` was a bit deceiving because it didn't indicate we are simply using random peer to pulll

Reply:
ok, I agree with @TheMMaciek then